Dockerfiles/catalog.Dockerfile

```
@@ -0,0 +1,22 @@
+# The builder image is expected to contain
+# /bin/opm (with serve subcommand)
+FROM quay.io/operator-framework/opm:latest as builder
+
+# Copy FBC root into image at /configs and pre-populate serve cache
+ADD catalog /configs
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)